misc/logging/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+use tracing_subscriber::EnvFilter;
+
+pub fn init_tracing_subscriber_with_default_env_filter() {
+    init_tracing_subscriber_with_env_filter(EnvFilter::from_default_env());
+}
+
+pub fn init_tracing_subscriber_with_env_filter(filter: impl Into<EnvFilter>) {
```

Comment:
Nit: can we shorten these names, e.g. just `with_{default_}env_filter`?

I think the test etc where we use this don't really need to know about the inner logging system. Especially because there are no alternatives to `tracing_subscriber` implemented in this crate.

Reply:
okay

Reply:
done